src/pages/home/home.html

```diff
@@ -99,7 +99,8 @@
           Access denied
         </div>
         <div class="server-message-body" translate>
-          Unfortunately, your country/region has banned the use or acceptance of cryptocurrencies as a valid form of payment and we are therefore unable to service you.
```

Comment:
keep this string in a single line so we don't have a new translation

Reply:
done, was auto html formatter from vscode